src/components/TaxPayer/SpouseAndDependent.tsx

```diff
@@ -211,15 +210,17 @@ const SpouseAndDependent = (): ReactElement => {
     <PagerContext.Consumer>
       { ({ onAdvance, navButtons }) =>
         <form onSubmit={handleSubmit(onSubmit(onAdvance))}>
-          <h2>Spouse Information</h2>
+          <h2>Family Information</h2>
+
+          <strong><p>Spouse Information</p></strong>
           <SpouseInfo />
 
-          <h2>Dependent Information</h2>
+          <strong><p>Dependent Information</p></strong>
```

Comment:
`<h2> or <h3>`?

Reply:
Can you elaborate on this comment? I'm not sure what it means

Reply:
Sorry, I just saw you removed an <h2> tag in favor of <strong><p>, seems like it should be a header tag, but it doesn't really matter.

Reply:
Oh yeah I removed <h2> in favor of <strong>  because I thought having one title <h2> and multiple <strong>  was a more clear layout. <h3> would definitely work as well. 

The reason I went with <strong> over <h3> is that I thought it might be cleaner to reuse the strongLabel logic rather than have a separate title